monitoring/tests/system/test_vpcsc_v3.py

```diff
@@ -119,22 +117,7 @@ def test_list_alert_policies(self):
         delayed_inside = lambda: client.list_alert_policies(name_inside)
         name_outside = client.project_path(PROJECT_OUTSIDE)
         delayed_outside = lambda: client.list_alert_policies(name_outside)
-        TestVPCServiceControlV3._do_test(delayed_inside, delayed_outside)
-
-    @pytest.mark.skipif(
-        PROJECT_INSIDE is None, reason="Missing environment variable: PROJECT_ID"
-    )
-    @pytest.mark.skipif(
-        PROJECT_OUTSIDE is None,
-        reason="Missing environment variable: GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT",
-    )
-    def test_update_alert_policy(self):
-        client = monitoring_v3.AlertPolicyServiceClient()
-        name_inside = client.alert_policy_path(PROJECT_INSIDE, "mock_alert_policy")
-        delayed_inside = lambda: client.update_alert_policy({"name": name_inside})
-        name_outside = client.alert_policy_path(PROJECT_OUTSIDE, "mock_alert_policy")
-        delayed_outside = lambda: client.update_alert_policy({"name": name_outside})
-        TestVPCServiceControlV3._do_test(delayed_inside, delayed_outside)
+        self._do_test(delayed_inside, delayed_outside)
```

Comment:
No particular reason. It is just because the file is auto-generated. I can sort the input endpoints to the script and make it deterministic from now on.